src/requirements/Sound/SoundRequirement.tsx

```diff
@@ -0,0 +1,157 @@
+              <Img src="/requirementLogos/sound.png" />
+            )
+        }
+      })()}
+      {...rest}
+    >
+      {(() => {
+        switch (requirement.type) {
+          case "SOUND_ARTIST":
+            return (
+              <>
+                {`Be an artist on `}
+                <Link
+                  href={`https://www.sound.xyz/`}
+                  isExternal
+                  colorScheme={"blue"}
+                  fontWeight="medium"
+                >
+                  Sound.xyz
+                </Link>
+              </>
+            )
+          case "SOUND_ARTIST_BACKED":
+            return (
+              <>
+                {`Support `}
+                <Link
+                  href={`https://www.sound.xyz/${requirement.data.id}`}
+                  isExternal
+                  fontWeight="medium"
+                  colorScheme={"blue"}
+                >
+                  {requirement.data.id}
+                </Link>
+                {` on `}
+                <Link
+                  href={`https://www.sound.xyz/`}
+                  isExternal
+                  fontWeight="medium"
+                  colorScheme={"blue"}
+                >
+                  Sound.xyz
+                </Link>
+              </>
+            )
+          case "SOUND_COLLECTED":
+            return (
+              <>
+                {`Own the `}
+                <Link
+                  href={`https://www.sound.xyz/${requirement.data.id}/${slugify(
+                    requirement.data.title,
+                    {
+                      lower: true,
+                      strict: true,
+                    }
+                  )}`}
+                  isExternal
+                  fontWeight="medium"
+                  colorScheme={"blue"}
+                >
+                  {requirement.data.title}
+                </Link>
+                {` song from `}
+                <Link
+                  href={`https://www.sound.xyz/${requirement.data.id}`}
+                  isExternal
+                  fontWeight="medium"
+                  colorScheme={"blue"}
+                >
+                  {requirement.data.id}
+                </Link>
+                {` on `}
+                <Link
+                  href={`https://www.sound.xyz/`}
+                  isExternal
+                  fontWeight="medium"
+                  colorScheme={"blue"}
+                >
+                  Sound.xyz
+                </Link>
+              </>
+            )
+          case "SOUND_TOP_COLLECTOR":
+            return (
+              <>
+                {`Be in the top 10 collector of `}
+                <Link
+                  href={`https://www.sound.xyz/${requirement.data.id}`}
+                  isExternal
+                  fontWeight="medium"
+                  colorScheme={"blue"}
+                >
+                  {requirement.data.id}
+                </Link>
+                {` on `}
+                <Link
```

Comment:
since the `on Sound.xyz` link is always there at the end, could just move it after the switch statement instead of repeating it